pages/docs/tutorials/validate.md

```diff
@@ -0,0 +1,105 @@
+
+To use the studio, you can update it with the code as shown below;
+
+```
+asyncapi: '2.2.0'
+info:
+  title: Validate App
+  version: '1.0.0'
+  description: Trying out the first steps to validating AsyncAPI.
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0'
+servers:
+  swamp:
+    url: localhost
+    protocol: ws
+channels:
+  user/login:
+    subscribe:
+      summary: A practice to login a user to a system.    
+      message:
+        $ref: '#/components/messages/UserLoginDetails'
+components:
+  messages:
+    UserLoginDetails:
+      payload:
+        type: object
+        properties:
+          userName:
+            type: string
+            description: A logged in user name.
+          password:
+            type: number
+            format: number
+            description: The logged in user password.
+```
+
+- To use the CLI, run;
+```
+npm install -g @asyncapi/cli
+```
+The code above helps you install the package globally on your system.
+
+- Then run the following command, follow the prompts to create a new specification file;
+```
+asyncapi new
+```
+- Open the file and make the necessary updates using the example specification file above or your own API definition.
+
+# Validation of AysncAPI Document
+When validating an AsyncAPI document, it could be in these forms;
+- Validation against the specification.
+- Validation against the company best practices or governance.
+
+### Validation against the specification
+ It entails making sure that every content of the document is written in accordance with the AsyncAPI specification.
+
+Using the AsyncAPI studio is a quick and easy way to validate against the specification.  It uses [AsyncAPI parser](https://github.com/asyncapi/parser-js) behind the scenes to perform syntax checks and see if a document is valid or not. 
+
+A red underline appears when an AsyncAPI document is invalid. The console provides feedback quickly, allowing you to check the format of your AsyncAPI document.
+
+In addition, the right-side of the studio is where changes made are seen and if a document is invalid, it gives the error `Empty or invalid document please fix errors / define AsyncAPI document`.
+
+
+### Validation against company best practices or governance
+When AsyncAPI is used by various members of the team in an organization, API management policies can be applied to your messages before they arrive to your broker.
+
+This can be done using **Spectral**, an API linting tool which has a built-in "asyncapi" ruleset for the AsyncAPI specification and helps to define best practices for your Event-Driven APIs. You can learn more about it [here](https://www.asyncapi.com/blog/creating-consistency-announcing-asyncapi-spectral-together).
+
+# Summary
+In this tutorial, we've learnt how to create an AsyncAPI specification file using either the studio or the CLI. We also learned about what validation of an AsyncAPI document entails using the studio.
+
+# Next Steps
```

Comment:
do we need it if this tutorial is last on the list?

Reply:
Urrm do you mean the next steps or summary, as both are part of the tutorial template outline

Reply:
I mean `next steps`. 

@alequetzalli if we need this them maybe we should point elsewhere, and not to the previous tutorial? 🤔

Reply:
Hmmm, can I point to Karuna's tutorial on message validation when it's merged?

Reply:
oh yes, definitely, great idea!

Reply:
Alright!